src/backend/joanie/tests/core/test_api_admin_enrollments.py

```diff
@@ -0,0 +1,540 @@
+"""Test suite for the admin enrollments API endpoints."""
+
+import uuid
+from http import HTTPStatus
+
+from django.test import TestCase
+from django.utils import timezone
+
+from joanie.core import enums, factories
+from joanie.core.models import CourseState
+from joanie.tests import format_date
+
+
+# pylint: disable=too-many-public-methods, too-many-lines
+class OrdersAdminApiTestCase(TestCase):
+    """Test suite for the admin enrollments API endpoints."""
+
+    maxDiff = None
+
+    def test_api_admin_enrollments_request_without_authentication(self):
+        """
+        Anonymous users should not be able to request enrollments endpoint.
+        """
+        response = self.client.get("/api/v1.0/admin/enrollments/")
+
+        self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
+        content = response.json()
+        self.assertEqual(
+            content["detail"], "Authentication credentials were not provided."
```

Comment:
According to our latest discussion with the team, it has been suggested to not assert what is done by DRF and just check the `status_code` of the `response`.